docs/source-2.0/aws/aws-endpoints-region.rst

```diff
@@ -412,3 +412,30 @@ The following example specifies a service that has standard regional endpoints e
      service MyService {
          version: "2020-04-02"
      }
+
+.. _aws.endpoints#endpoint-pattern:
+
+----------------
+Endpoint Pattern
```

Comment:
This should declare that patterns SHOULD begin with a scheme of either `http` or `https`.